src/EditorFeatures/Test2/Rename/RenameEngineResult.vb

```diff
@@ -55,7 +56,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             workspace.SetTestLogger(AddressOf helper.WriteLine)
 
             workspace.TryApplyChanges(workspace.CurrentSolution.WithOptions(
-                workspace.Options.WithChangedOption(RemoteHostOptions.RemoteHostTest, host = TestHost.OutOfProcess)))
+                workspace.Options.WithChangedOption(RemoteHostOptions.RemoteHostTest, host <> TestHost.InProcess)))
```

Comment:
@ryzngard This is an example of what i was talking about earllier.  We want OOP enabled for all optoins that are not 'InProcess'.

Reply:
Not sure I follow. Are there multiple `OutOfProcess` values? It seems like you're basically comparing `value != false` instead of `value == true`  (ignoring that value could just be the argument here...)

Reply:
Reading further now I get it 

```
    Public Enum TestHost
        InProcess	        InProcess
        OutOfProcess	        ' Work out of process, marshaling to/from RenameSymbolAsync
        OutOfProcess_SingleCall
        ' Work out of process, marshaling to/from FindRenameLocations, then marshaling to/from ResolveConflictsAsync
        OutOfProcess_SplitCall
    End Enum	    End Enum
``` 

There are multiple out of process hosts. That's the missing bit :)